Remotion/Security/Core/Metadata/EnumerationReflector.cs

```diff
@@ -64,13 +65,14 @@ public EnumValueInfo GetValue (Enum value, MetadataCache cache)
       ArgumentUtility.CheckNotNull ("value", value);
       ArgumentUtility.CheckNotNull ("cache", cache);
 
-      EnumValueInfo info = cache.GetEnumValueInfo (value);
+      EnumValueInfo? info = cache.GetEnumValueInfo (value);
       if (info == null)
       {
         string name = value.ToString ();
         info = new EnumValueInfo (TypeUtility.GetPartialAssemblyQualifiedName (value.GetType ()), name, Convert.ToInt32 (value));
-        FieldInfo fieldInfo = value.GetType ().GetField (name, BindingFlags.Static | BindingFlags.Public);
-        PermanentGuidAttribute attribute = (PermanentGuidAttribute) Attribute.GetCustomAttribute (fieldInfo, typeof (PermanentGuidAttribute), false);
+        // TODO: Add notnull assertion
+        FieldInfo fieldInfo = value.GetType ().GetField (name, BindingFlags.Static | BindingFlags.Public)!;
```

Comment:
that's a debug-notnull at most. The value cannot be null given how .net works. Ticket or update